public/js/dom.js

```diff
@@ -0,0 +1,44 @@
+// const url = `https://api.themoviedb.org/3/search/movie?api_key=${config.apiKey }&language=en-US&query=`;
+function selecElement(id) {
+	return document.querySelector(id);
+}
+function addEventListenerForSelector(selectorElement, event, callback){
+	return document.querySelector(selectorElement).addEventListener(event,callback);
+}
+function createElement(e){
+	return document.createElement(e);
+}
+addEventListenerForSelector('#submit-btn', 'submit', (event) => {
+	event.preventDefault();
+	if (selecElement('#searchInput').value == ""){
+		alert('Entar a movie name');
+	}
+	selecElement('#searchInput').innnerHTML = "";
+	var url = `https://api.themoviedb.org/3/search/movie?api_key=${config.apiKey }&language=en-US&query= + ${selecElement('#searchInput').value}`;
+	fetchFunction('GET', url, (response) => {
+		var result = response.results;
+		cosole.log(result);
+		if (result.length == 0){
+			var msg = createElement('p').innnerHTML = "No Results";
```

Comment:
Weird code  
the paragraph will not change it's content 
this line of code like this exactly : "nod"